apps/dashboard/src/components/workflow-editor/steps/configure-step-form.tsx

```diff
@@ -64,10 +65,6 @@ export const ConfigureStepForm = (props: ConfigureStepFormProps) => {
 
   const form = useForm<z.infer<typeof stepSchema>>({
     defaultValues,
-    values: {
-      ...defaultValues,
-      ...step.controls.values,
-    },
```

Comment:
no need to use `values` here, the form should always be set with default values that should have the latest step changes